broad-wb-malkov.json

```diff
@@ -1,64 +1,91 @@
 {
   "name": "terra",
   "owner": "gmalkov",
-  "terraform_image": "us.gcr.io/broad-dsp-gcr-public/config-render:48229e65030",
   "intent": "terra infrastructure",
   "production": false,
   "stability": "volatile",
+  "environment": "broad-wb-malkov",
+  "project": "broad-wb-malkov",
+  "region": "us-central1",
+  "resource_zone_name": "us-central1-a",
+  "dns_zone_name": "dsde-perf-broad",
+  "dns_zone": "dsde-perf.broadinstitute.org",
   "global_vars": {
-    "env": {
-      "VAULT_PATH_PREFIX": "secret/dsde/firecloud/ephemeral/broad-wb-malkov"
-    },
     "vault": {
       "policy": "dsde-write"
+    },
+    "env": {
+      "VAULT_PATH_PREFIX": "secret/dsde/firecloud/ephemeral/gmalkov"
+    },
```

Comment:
could you generate the `VAULT_PATH_PREFIX` programmatically based on owner? `"secret/dsde/firecloud/ephemeral/${owner}"`? I suppose users could override this if necessary.